Lib/venv/__init__.py

```diff
@@ -178,6 +178,8 @@ def create_configuration(self, context):
             f.write('version = %d.%d.%d\n' % sys.version_info[:3])
             if self.prompt is not None:
                 f.write(f'prompt = {self.prompt!r}\n')
+            f.write('executable = %s\n' % context.env_exec_cmd)
+            f.write('command = %s\n' % sys.executable)
```

Comment:
That (`sys.executable`) is not the command (-line) which was used to create the environment - it would be the  `python -m venv ...` or equivalent command line which need to be recorded.

Reply:
Would something like this work?

```
            args = ' '.join(sys.argv[1:])
            f.write(f'command = {sys.executable} -m {context.env_name} {args}\n')
```

Reply:
> Would something like this work?

Not necessarily. The code to create a venv might be called programmatically through the `EnvBuilder` API. The intention is to provide a command which, if run, produces a venv with the current settings. So the thing to do is:
1. Run `python3 -m venv -h` to see what the command-line options are.
2. Look at the existing code to see how those options map to attributes of the `EnvBuilder` which are used in making the venv.
3. Looking at the `EnvBuilder` attributes of the current instance, work out what command-line would give those attributes, and synthesize the command line from there.